content/learning-paths/smartphones-and-mobile/get-started-with-unity-on-android/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Get started with Unity on Android
+
+minutes_to_complete: 45
```

Comment:
LP3 justifies its high reading time! In theory too long for a learning path? Although I don't have a suggestion as to how it could possibly be split. Maybe some of the Neon intrinsics explanation detail could be an appendix or something, so the bulk is reduced for the Learning Path proper?